src/test/system_tests/test_utils.js

```diff
@@ -335,7 +335,40 @@ async function delete_fs_user_by_platform(name) {
     }
 }
 
-/** 
+/**
+ * creates a new file system group. if user_name is define add it to the group
+ * @param {string} group_name
+ * @param {number} gid
+ * @param {string} user_name
+ */
+async function create_fs_group_by_platform(group_name, gid, user_name) {
```

Comment:
Do you care only about the platform (Linux/MacOS)? Running on another FS would be fine? (GPFS for example)

Reply:
GPFS is a linux platform